### DISCUSSION TITLE
Standalone visual editor?

### DISCUSSION THREAD

**Discussion by ratnanil** [quarto-dev/quarto-cli#3948] [Q&A]
Ever since I've seen this following sentence on [the quarto website](https://quarto.org/docs/visual-editor/#getting-started), I check it up regularly to see if there is an update on this:

> The visual editor will eventually also be made available in standalone form.

I particularly am looking forward to this feature, since I preach the use of markup languages for many (non-coding) use cases for non coders in our institution. Having a lightweight IDE detached from R / Python etc would greatly help in this mission.

**Comment by mcanouil** [comment]
No worries, far as I know it's definitely planned, although I don't think it will come in early 2023, but who knows 🤷‍♂️

**Reply by reagle** [reply]
Is this still in the works? (Asking 18 months later.)

**Reply by mcanouil** [reply]
Still planed but nothing to be announced.
